api/__init__.py

```diff
@@ -0,0 +1,92 @@
+"""Backend api server that is used in the MOH project.
+
+A simple flask backend that will maintaine a live queue of current students in office hours and keep
+track of current student status in the within the queue
+by making request updating the `vist` table within the external postgrest database.
```

Comment:
```suggestion
track of current student status within the queue
by making request updating the `vist` table within the external Postgres database.
```

I'm not sure what the second line is trying to convey, or if the table is actually named "vist".